README.md

```diff
@@ -54,6 +54,14 @@ const result = await appAuth.authorize(scopes);
 // returns accessToken, accessTokenExpirationDate and refreshToken
 ```
 
+#### config
+
+This is your configuration object for the client
+- **issuer**: (`string`) *REQUIRED* the url of the auth server
+- **clientId**: (`string`) *REQUIRED* your client id on the auth server
+- **redirectUrl**: (`string`) *REQUIRED* the url that links back to your app with the auth code
+- **additionalParameters**: (`object` | `null`) additional parameters that will be passed in the authorization request
```

Comment:
Would it be helpful to document how these additional parameters are passed in the request or is that obvious to anyone who is using this?

Reply:
Good point. Essentially

```
{ hello: 'world` }
```

would be sent as

```
<url-and-other-params>&hello=world
```

But I'll make it more explicit in the docs.